next.config.js

```diff
@@ -34,6 +34,15 @@ const nextConfig = {
     styledComponents: true,
   },
 
+  async rewrites() {
+    return [
+      {
+        source: '/glry/:path*',
+        destination: `${process.env.NEXT_PUBLIC_API_BASE_URL}/glry/:path*`,
+      },
+    ];
+  },
```

Comment:
This basically says if we hit `gallery.so/glry/graphql/query`, Vercel will pipe the request to `api.gallery.so/glry/graphql/query`